hopp/simulation/technologies/steel/enthalpy_functions.py

```
@@ -0,0 +1,194 @@
+    H_t=(A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H)/mol_weight_fe
+    return H_t
+def feo_enthalpy(T):
+    mol_weight_feo=71.844 #in grams
+    t=T/1000
+    A=45.75120
+    B=18.78553
+    C=-5.952201
+    D=0.852779
+    E=-0.081265
+    F=-286.7429
+    G=110.3120
+    H=-272.0441
+    H_t=(A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H)/mol_weight_feo
+    return H_t
+def al2o3_enthalpy(T):
+    mol_weight_al2o3=101.9613 #in grams    
+    t=T/1000
+    A=106.0880
+    B=36.33740
+    C=-13.86730
+    D=2.141221
+    E=-3.133231
+    F=-1705.970
+    G=153.9350
+    H=-1662.300  
+    H_t=(A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H)/mol_weight_al2o3
+    return H_t
+def sio2_enthalpy(T):
+    mol_weight_Sio2=60.0843 #in grams
+    t=T/1000
+    A=58.75
+    B=10.279
+    C=-0.131384
+    D=0.025210
+    E=0.025601
+    F=-929.3292
+    G=105.8092
+    H=-910.8568   
+    H_t=(A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H)/mol_weight_Sio2
+    return H_t
+def mgo_enthalpy(T):
+    mol_weight_mgo=40.3044 #in grams
+    t=T/1000
+    A=47.25995
+    B=5.681621
+    C=-0.872665
+    D=0.104300
+    E=-1.053955
+    F=-619.1316
+    G=76.46176
+    H=-601.2408
+    H_t=(A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H)/mol_weight_mgo
+    return H_t
+def cao_enthalpy(T):
+    mol_weight_cao=56.077#in grams
+    t=T/1000
+    A=49.95403
+    B=4.887916
+    C=-0.353056
+    D=0.046187
+    E=-0.825097
+    F=-652.9718
+    G=92.56096
+    H=-635.0894
+    H_t=(A*t +(B*t*t)/2 +(C*t*t*t)/3 + (D*t*t*t*t)/4-(E/t)+F-H)/mol_weight_cao
+    return H_t
+def ch4_enthalpy_1(T):# T in the range 298-1000 K
+    mol_weight_CH4=16.04 # in grams
+    t=T/1000
+    A=-0.703029
+    B=108.4773
+    C=-42.52157
+    D=5.862788
+    E=0.678565
+    F=-76.84376
+    G=158.7163
+    H=-74.87310
+    H_t=(A*t +(B*t**2)/2 +(C*t**3)/3 + (D*t**4)/4-(E/t)+F-H)/mol_weight_CH4
+    return H_t
+def ch4_enthalpy_2(T):# T in range 1300-6000 K
```

Should also combine the ch4 functions. You may want to throw a value error in case a temperature given is out of range (same for other temperature input functions here). That is a lot safer than just a comment.

---

Combined ch4 functions as well